src/modules/core/components/Fields/Select/Select.css

```diff
@@ -88,6 +88,7 @@
   border: none;
   border-radius: var(--radius-normal);
   background-color: transparent;
+  outline: 0;
```

Comment:
Please **don't remove** the `outline` on the core component directly. _(or other accessibility features for that matter)_

If you still want to get rid of it, do it in your local component.